lib/routemap_northbound.c

```diff
@@ -281,7 +282,17 @@ static int lib_route_map_entry_action_modify(struct nb_cb_modify_args *args)
 	case NB_EV_APPLY:
 		rmi = nb_running_get_entry(args->dnode, NULL, true);
 		rmi->type = yang_dnode_get_enum(args->dnode, NULL);
-		/* TODO: notify? */
+		map = rmi->map;
+
+		route_map_pfx_tbl_update(RMAP_EVENT_INDEX_ADDED, rmi, 0, NULL);
```

Comment:
I'm not sure this is needed as there was no previous call for `RMAP_EVENT_INDEX_DELETED`. Could you remove this call and check that the update still works?

Reply:
agree . Addressed .